README.md

```diff
@@ -68,6 +70,12 @@ Join mailing list:
 **The Ethics Centre**  
 [https://ethics.org.au/](https://ethics.org.au/)  
 
+**Data and Society**
+[https://datasociety.net/](https://datasociety.net/)
+
+**Center for Humane Technology**
+[https://www.humanetech.com/](https://www.humanetech.com/)
```

Comment:
So, um, TL;DR I'm not a fan of Tristan Harris and will not promote his group, could we remove them?  

Happy to have a longer conversation about this (and open to your feedback if you have a strong argument that there is something redeemable here). Without writing a novel, you can find more in depth critique https://www.youtube.com/watch?v=Xazea6_Noks&ab_channel=TheNewDesignCongress and https://conversationalist.org/2020/03/05/the-prodigal-techbro/